.github/workflows/ci-turbo-build.yml

```diff
@@ -26,10 +26,17 @@ jobs:
       # precompiled binary isn't found.
       - name: Install libusb
         run: sudo apt-get update && sudo apt-get install -y libusb-1.0-0-dev libudev-dev
+      # Install Foundry for Ethereum contract builds
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: v0.3.0
       - uses: pnpm/action-setup@v4
```

Comment:
github workflows is updated as now part of build/test we will run forge commands.